protocol/contracts/beanstalk/barn/UnripeFacet.sol

```diff
@@ -189,19 +182,11 @@ contract UnripeFacet is ReentrancyGuard {
         view
         returns (uint256 penalty)
     {
-        return getRecapPaidPercentAmount(getRecapFundedPercent(unripeToken));
+        return LibChop.getRecapPaidPercentAmount(getRecapFundedPercent(unripeToken));
     }
-
+    
     function getRecapPaidPercent() external view returns (uint256 penalty) {
-        penalty = getRecapPaidPercentAmount(DECIMALS);
-    }
-
-    function getRecapPaidPercentAmount(uint256 amount)
-        private
-        view
-        returns (uint256 penalty)
-    {
-        return s.fertilizedIndex.mul(amount).div(s.unfertilizedIndex);
+        penalty = LibChop.getRecapPaidPercentAmount(DECIMALS);
```

Comment:
can we use `DECIMALS` in `LibUnripe` rather than having a new constant variable in `UnripeFacet.sol`?

Reply:
Good point. Fixed in: https://github.com/BeanstalkFarms/Beanstalk/pull/460/commits/5fe2485c2659fd347a0af43b718fd18e6f3b0bdb

Reply:
great